benches/benches/root.rs

```diff
@@ -7,6 +7,10 @@ mod renderpass;
 mod resource_creation;
 mod shader;
 
+fn is_test() -> bool {
+    std::env::var("NEXTEST").is_ok()
```

Comment:
question: Do we want to check the contents of the `NEXTEST` variable, so, i.e., `NEXTEST=false` doesn't actually trigger this?

Reply:
I don't expect people to ever set this manually, even accidentally, only by cargo-nextest, so it's probably _vaguely_ safer to not tightly couple to the exact value